src/color.h

```diff
@@ -80,13 +80,15 @@ typedef unsigned char ColorType;
 
 extern unsigned long colors[COLOR_COUNT];
 
+
 /*@{*/
 #define InitializeColors() (void)(0)
 void StartupColors(void);
 void ShutdownColors(void);
 void DestroyColors(void);
 /*@}*/
 
+
```

Comment:
Purely a style thing, but we probably don't need these extra lines in color.h and color.c (especially since extra lines are the only change).

Reply:
I don't put this line, they are in original source code, probably git identify them as changed because initially i removed them then re-insert in the same class.